src/sipjs-card.ts

```diff
@@ -416,6 +416,12 @@ class SipJsCard extends LitElement {
             case "scene":
                 service = "turn_on";
                 break;
+            case "light":
+                service = "toggle";
+                break;
+            case "homeassistant":
```

Comment:
Homeassistant isn't a entity. Did you mean switch?

```suggestion
            case "switch":
```

Reply:
good pint!